linux/net/rina/rmt.h

```diff
@@ -59,6 +59,7 @@ struct rmt;
 enum flow_state {
 	N1_PORT_STATE_ENABLED,
 	N1_PORT_STATE_DISABLED,
+	N1_PORT_STATE_DO_NOT_DISABLE,
```

Comment:
Moreover, imho the first person that should ACK/NACK the code based on design aspects is the maintainer of the component, since he/she is the person that knows more about the design of that particular component and about the expected usage of its data structures and functions. 

Otherwise we are defeating one of the points of having maintainers, in my opinion.